src/kohlrahbi/__init__.py

```diff
@@ -259,11 +266,11 @@ def process_pruefi(
     for ahb_file_path in ahb_file_paths:
         doc = get_or_cache_document(ahb_file_path, path_to_document_mapping)
```

Comment:
ist in dem Repo out of scope, aber die Funktion tut zwei Dinge was sie auch im Namen schon verrät.
Wäre cooler wenn das entsprechend zwei Funktionen sind. Aber das ist für einen anderen PR ^^